examples/frustum-cull/app.js

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable no-continue */
+/* global requestAnimationFrame, cancelAnimationFrame */
+
+import React, {Component} from 'react';
+import {render} from 'react-dom';
+import DeckGL, {SimpleMeshLayer} from 'deck.gl';
+import {StaticMap} from 'react-map-gl';
+import {SphereGeometry} from '@luma.gl/core';
+import {Vector3} from 'math.gl';
+
+const INITIAL_VIEW_STATE = {
+  latitude: 51.47,
+  longitude: 0.45,
+  zoom: 7,
+  bearing: 0,
+  pitch: 0
+};
+
+const position = [-0.4531566, 51.4709959, 100000];
+const radius = 10000;
+const testPosition = new Vector3();
+
+class Root extends Component {
+  constructor(props) {
+    super(props);
+    this.deckRef = React.createRef();
+    this.animationLoop = null;
+    this.state = {
+      cullStatus: 'IN'
+    };
+
+    this.statusStyle = {
+      position: 'absolute',
+      right: '10px',
+      top: '10px',
+      padding: '10px',
+      background: 'white',
+      border: '1px solid black'
+    };
+
+    this.loop = this.loop.bind(this);
+  }
+
+  componentDidMount() {
+    this.animationLoop = requestAnimationFrame(this.loop);
```

Comment:
You should be able to subscribe to `onViewStateChange` instead of creating an animation loop